ona/tasks.py

```diff
@@ -0,0 +1,69 @@
+from celery.task import Task
+from celery.utils.log import get_task_logger
+from celery.exceptions import SoftTimeLimitExceeded
+from django.core.exceptions import ObjectDoesNotExist
+import requests
+from requests.exceptions import HTTPError
+from .models import Submission
+
+logger = get_task_logger(__name__)
+
+
+class Send_Submission(Task):
+
+    """
+    Task to load and construct submission and send it off
+    """
+    name = "ona.tasks.send_submission"
+
+    class FailedEventRequest(Exception):
+
+        """
+        The attempted task failed because of a non-200 HTTP return code.
+        """
+
+    def run(self, submission_id, **kwargs):
+        try:
+            submission = Submission.objects.get(pk=submission_id)
+            if submission.submitted is False:
+                integration = submission.integration.details
+                data = '{"submission": '+submission.content+', "id": \
+                    "'+integration["form_id"]+'"}'
```

Comment:
Is this data JSON? In that case we should do `import json` and `data = json.dumps({"submission": submission.content, "id": integration["form_id"]})` rather than constructing the JSON ourselves (since the content might contain quotes or newlines of unicode characters, etc).